src/app/pages/OnDemandTvPage/VideoPlayer/index.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { BrowserRouter } from 'react-router-dom';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import VideoPlayer from '.';
```

Comment:
Should `VideoPlayer` live within `#containers` rather than the page folder? The only other example of this pattern is `MediaMessage` within `MediaAssetPage`, which was us and probably _very_ naughty.

Reply:
I very much prefer this pattern to be honest - it's just grouping by feature

Components should be located close to where they're being used, and this component is only being used on this one page

I'd definitely recommend moving it out of here if it ever gets used in more than one place though